models/comment.js

```diff
@@ -1,9 +1,37 @@
 const mongoose = require('../mongoose');
+const Schema = mongoose.Schema;
 const uuid = require('uuid');
 const Action = require('./action');
 
-const Schema = mongoose.Schema;
+/**
+ * The Mongo schema for a Comment Status.
+ * @type {Schema}
+ */
+const StatusSchema = new Schema({
```

Comment:
Could we add a field in here to store the kind of action that was taken in order to set this status:

* "moderator" - an action explicitly taken by a moderator
* "blacklist" - a status set because of a match against the blacklist
* "" - default setting

We could possibly used the assigned_by field to capture this information, however I wouldn't want to use a single field to capture both a moderator's id and something like "blacklist"